source/priority-queue.ts

```diff
@@ -0,0 +1,44 @@
+import {Queue, RunFunction} from './queue';
+import lowerBound from './lower-bound';
+import {QueueAddOptions} from './options';
+
+export interface PriorityQueueOptions extends QueueAddOptions {
+	priority: number;
+}
+
+export default class PriorityQueue implements Queue<PriorityQueueOptions> {
+	private readonly queue: (PriorityQueueOptions & { run: RunFunction })[];
+
+	constructor() {
+		this.queue = [];
+	}
+
+	enqueue(run: RunFunction, opt?: PriorityQueueOptions): void {
```

Comment:
`opt` => `options`

Reply:
Your tslint config disagreed with overwriting the incoming object, reverting it now ✅👍